__tests__/colors.js

```diff
@@ -1,19 +0,0 @@
-import mapObject from 'map-obj'
-import isObject from 'is-object'
-
-import * as colors from '../src/colors.js'
-
-const removeValues = (obj) => mapObject(obj, (key, value) => ([key, '']), { deep: true })
-
-test('colors', () => {
-	expect(colors).toMatchSnapshot()
-})
-
-test('keys equal', () => {
```

Comment:
Maybe would be worth to keep this test so you are sure that light, dark and mirage always have the same keys. The other test is stupid and was just that I can quickly see what changed.